.circleci/config.yml

```
@@ -23,4 +23,4 @@ jobs:
       - when:
           condition: <<pipeline.git.tag>>
           steps:
-            - run: gh release upload --repo lovell/sharp-libvips $CIRCLE_TAG *.tar.gz *.integrity
+            - run: gh release upload --repo "$(git config --get remote.origin.url)" $CIRCLE_TAG *.tar.gz *.integrity
```

Note to self: I considered using the new [`/upload`](https://circleci.com/developer/orbs/orb/circleci/github-cli#jobs-upload) job here, e.g.:
```suggestion
            - github-cli/upload:
                files: "*.tar.gz *.integrity"
```

But that job seems to do a `gh repo clone`, so it probably won't work on the recently build files.
https://github.com/CircleCI-Public/github-cli-orb/blob/e89b4c51027a6659a54235d0d3358f7ff1145494/src/jobs/upload.yml#L45